<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.584, y1:0.466045, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(153, 0, 255, 255));</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0.3)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="email_line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: rgb(255,255,255);
border-color:rgb(255,255,255);
border-width: 1px; /* Define a largura da borda para 2 pixels */
border-style: solid;
border-radius:5px; /* Define o estilo da borda como sólido */
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="senha_line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>300</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: rgb(255,255,255);
border-color:rgb(255,255,255);
border-width: 1px; /* Define a largura da borda para 2 pixels */
border-style: solid;
border-radius:5px; /* Define o estilo da borda como sólido */
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_entrar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0,0,0);
color:rgb(255,255,255);
border-radius: 10px;
}

QPushButton:hover {
background-color: rgb(255,255,255);
color: rgb(0,0,0);
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">        QFrame {
            background-color: rgba(231, 231, 231, 0); /* Define a cor de fundo como transparente */
        }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="logo_.qrc">:/logo/pages/icons/logo_nome_branco.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logo_.qrc"/>
 </resources>
 <connections/>
</ui>
